package.json

```diff
@@ -25,6 +25,7 @@
     "lint:fix": "eslint --fix -f ./node_modules/eslint-friendly-formatter \"{,**/}*.{js,vue}\" && stylelint \"**/*.{css,vue}\" --fix",
     "test": "yarn run lint && yarn test:unit && yarn test:e2e",
     "test:unit": "cross-env NODE_ENV=testing jest --maxWorkers=2",
+    "test:updateSnapshot": "cross-env NODE_ENV=testing jest --updateSnapshot",
```

Comment:
you should probably do this on a file by file basis. `jest PageValidator -u`

Reply:
so this should probably removed to not confuse developers to use it.

Reply:
OK sounds good. Thanks!